### DISCUSSION TITLE
self-hosted community version default account

### DISCUSSION THREAD

**Discussion by threatseeker** [Syslifters/sysreptor#128] [General]
I set it up my own ubuntu VM to take the sysreptor for a spin. After installing it, the local page http://127.0.0.1:8000/ prompts for a login, what is the default local user account name and password? (self-hosted community version install).  Not sure if I missed a step?

**Comment by aronmolnar** [comment]
Hi, username and password are displayed at the end of the installation script.
If you don't have it any longer, you can go to the deploy directory and change it using `docker compose exec app python3 manage.py changepassword reptor`. You can then set a new password and login with username reptor.
